floss/qs/db/expert.py

```diff
@@ -51,13 +49,13 @@ def query(self, s: str) -> Set[str]:
         return ret
 
     @classmethod
-    def from_file(cls, path: pathlib.Path) -> "ExpertStringDatabase":
+    def from_file(cls, package: str, resource: str) -> "ExpertStringDatabase":
```

Comment:
let's keep `from_file` around since it's really useful for dev and testing. but let's add a `from_pkgutil` to address the feature request. and please factor out the common code into a sub routine.

Reply:
Which routine will the `get_default_databases` use for now ?

Reply:
please prefer pkgutil